### PR TITLE
Locales, fix, trunk size

### PR DESCRIPTION
**Pull request by TonybynMp4**
*Localized everything (FR + EN)
*TP player in managed test drive
*Added trunk size in headers (in accordance to my Core / Inventory PRs, can be removed easily if needed)
/!\ Because of the trunk size you'll need both my PRs to test it without any problems

Questions (please complete the following information):

Have you personally loaded this code into an updated qbcore project and checked all it's functionality?
I didn't test the command since im lonely (tried giving to myself but it literally did nothing, even outside the vehicle, probably already bugged ?) but the rest should be fine
Does your code fit the style guidelines?
yes
Does your PR fit the contribution guidelines?
yes

### CODE REVIEW

**Comment by GhzGarage** [comment]
Please fix conflicts and remove trunk code

File: client.lua [hunk 1]
```diff
@@ -620,38 +625,42 @@ RegisterNetEvent('qb-vehicleshop:client:getVehicles', function()
                 }
             end
         end
-        exports['qb-menu']:openMenu(ownedVehicles)
+              if #ownedVehicles > 0 then
```
Comment (BerkieBb) [inline client.lua:628]: This indentation needs to move 4 spaces or 1 tab back

**Review by BerkieBb — CHANGES_REQUESTED** [review]

File: client.lua [hunk 2]
```diff
@@ -402,28 +406,29 @@ RegisterNetEvent('qb-vehicleshop:client:customTestDrive', function(data)
         local vehicle = data
         local prevCoords = GetEntityCoords(PlayerPedId())
         QBCore.Functions.SpawnVehicle(vehicle, function(veh)
+            TaskWarpPedIntoVehicle(PlayerPedId(data.playerid), veh, -1)
```
[the remark this answers is not in the document]
Comment (TonybynMp4) [reply client.lua]: Done, i didn't know that thanks :p

File: client.lua (hunk 1 above)
Comment (TonybynMp4) [reply client.lua]: 👍🏻

**Comment by TonybynMp4** [comment]
> Please fix conflicts and remove trunk code

should be all good now

**Pull request merged**